api/v1alpha1/paasconfig_types.go

```diff
@@ -35,11 +35,14 @@ const (
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:path=paasconfig,scope=Cluster
 type PaasConfig struct {
-	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
+	metav1.TypeMeta `json:",inline"`
+	// +kubebuilder:validation:Optional
```

Comment:
As represented in the generated manifest, this has no effect nor is needed. Please don't add this here.

Reply:
I looked into it, and it seems we can either use omitempty, or `// +kubebuilder:validation:Optional`.
As a project it seems we use omitempty instead of `// +kubebuilder:validation:Optional`.
We can change that, but for now I have reverted all `// +kubebuilder:validation:Optional` tags to omitempty flags.